TicTacToeApp/ComputerPlayer.swift

```diff
@@ -7,6 +7,16 @@ public class ComputerPlayer {
   }
 
   public func makeMove(gameState: Game) -> Int {
+    var move: Int?
+    let serviceUrl = "http://107.170.25.194:5000/"
+    let request = computerMoveRequest(gameState, baseUrl: serviceUrl)
+
+    makeBlockingRequest(request, onSuccess: { (data) in move = Int(data) })
+
+    return move ?? getRandomElement(availableSpaces(gameState))
```

Comment:
Aside from the Reset Game and Retry Computer Move options outlined in this iteration's stories? Maybe the option to preserve game state and switch over to a human vs. human game. What do you have in mind? I agree with you about removing getRandomElement(). Since it's not a refactor, however, I'm going to wait until the first error handling story to remove it.